docs/source/02_installation.md

```diff
@@ -1,4 +1,4 @@
-# Building ADAM from Source {#build-from-source}
+# Building ADAM from Source
```

Comment:
Ah, I was just going by what was on adam.readthedocs.io, I didn't run docs/build.sh. The updated form does work with Sphinx, I tested them locally. But I can revert all my anchor tag/anchor link changes for now then. No need to change the format if links work.